src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -690,7 +690,14 @@ std::string postUpdateContext
     if (fails == response.contextElementResponseVector.size())
     {
       // If all CER result in error, then it isn't a partial update, but a regular NotFound
-      parseDataP->upcrs.res.oe.fill(SccContextElementNotFound, "No context element found", "NotFound");
+      if (ciP->apiVersion == "v1")
+      {
+        parseDataP->upcrs.res.oe.fill(SccContextElementNotFound, "No context element found", "NotFound");
```

Comment:
nit: third parameter is not needed for v1 (althought it doesn't hurt, anyway...)